src/tcgwars/logic/impl/gen3/HolonPhantoms.groovy

```diff
@@ -338,7 +338,20 @@ public enum HolonPhantoms implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 30
-            reduceDamageNextTurn(hp(30), thisMove)
+            afterDamage{
```

Comment:
What does this do differently than `reduceDamageNextTurn`?

Reply:
reduceDamage does the reduction after weakness and resistance is applied. Since there's not a static that handles it prior to that, it had to be hardcoded in the card.